code/game/gamemodes/vampire/vampire.dm

```diff
@@ -275,6 +275,9 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 	var/blood = 0
 	var/old_bloodtotal = 0 //used to see if we increased our blood total
 	var/old_bloodusable = 0 //used to see if we increased our blood usable
+	if(owner.is_muzzled())
+		to_chat(owner, "<span class='warning'>The muzzle prevents you from biting!</span>")
```

Comment:
You should change the `is_muzzled` proc to return an item reference if there is infact a muzzle on the mob, and then use that item reference to change this to_chat into `to_chat(owner, "<span class='warning'>[I] prevents you from biting [H]!</span>")`.